package.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>nps_uw_sensors</name>
+  <version>0.0.1</version>
+  <description>
+    NPS Gazebo underwater sensor plugins.
+  </description>
+
+  <maintainer email="bdallen@nps.edu">Bruce Allen</maintainer>
+  <author email="bdallen@nps.edu">Bruce Allen</author>
+  <license>Apache-2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>gazebo_dev</build_depend>
+  <exec_depend>gazebo_dev</exec_depend>
+
+  <depend>gazebo_msgs</depend>
```

Comment:
I don't think we need all these dependencies - do we?

Reply:
No.  How can I tell which ones are required?  I can remove them all and it works, but if someone has a system that does not have a requisite dependency installed, it will fail.  We use this to find requirements early.

Reply:
I would start with the header files and the includes in the source.  We could setup a CD/CI pipeline to test with a docker image.  That would be a good way to spot dependency issues.  

I put a trello card up for this, but for now just take try and do it 'by hand'.  We can always fix something if we find a missing dependency, but we shouldn't include a bunch of elements we don't use.

Reply:
Estimated without using Docker image to test.